src/util/Path.cpp

```diff
@@ -272,29 +249,29 @@ GFile* Path::toGFile()
 /**
  * Get the parent path
  */
-Path Path::getParentPath()
+Path Path::getParentPath() const
 {
 	size_t separator = path.find_last_of("/\\");
 
 	if (separator == string::npos)
 	{
-		return "";
+		return Path{""};
```

Comment:
I don't know this syntax, what are the advantages of this?

Reply:
That's an explicit initializer, it replaced the old c/java styled () - braced style. Its deprecated to use those () braces or Type varname = Type(vars). But actually are no differences to Path("");

The correct way to initialize variables is:
Type typename{...};
for return Type{...}
in c++17 return {...}